backend/unitTests/loginTest.py

```
@@ -0,0 +1,75 @@
+def test_admin_login():
+    """Test default admin login"""
+    admin_data = {
+        "username": "admin",
+        "password": "password123"
+    }
+    
+    try:
+        response = requests.post(f"{BASE_URL}/api/auth/login", json=admin_data)
+        print(f"Admin Login Status: {response.status_code}")
+        print(f"Admin Login Response: {response.json()}")
+        return response.status_code == 200
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)